app/pages/HomePage/components/BreakingNews/BreakingNews.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  CarouselProvider,
+  Slider,
+  Slide,
+  ButtonBack,
+  ButtonNext,
+} from 'pure-react-carousel';
+
+import styles from './BreakingNews.module.scss';
+
+const BreakingNews = () => (
```

Comment:
I left a similar comment on https://github.com/ShelterTechSF/askdarcel-web/pull/1168#discussion_r1112604637, but could we name this NewsArticle to make it consistent between the frontend and backend? Or if it's acceptable, renaming the backend API routes and database table could also be acceptable. I just think it'll cause confusion to people reading the code if the frontend and backend call this something different.